suites/pacific/cephfs/tier_1_fs.yaml

```diff
@@ -70,57 +70,23 @@ ts:
         config:
             command: add
             id: client.1
-            node: node2
+            node: node8
             install_packages:
                 - ceph-common
             copy_admin_keyring: true
         desc: Configure the Cephfs client system 1
         destroy-cluster: false
         module: test_client.py
         name: configure client
-  - test:
-      abort-on-fail: true
-      config:
-        command: add
-        id: client.2
-        node: node3
-        install_packages:
-          - ceph-common
-        copy_admin_keyring: true
-      desc: Configure the Cephfs client system 2
-      destroy-cluster: false
-      module: test_client.py
-      name: configure client
-
-  - test:
-      abort-on-fail: true
-      config:
-        command: add
-        id: client.3
-        node: node1
-        install_packages:
-          - ceph-common
-        copy_admin_keyring: true
-      desc: Configure the Cephfs client system 3
-      destroy-cluster: false
-      module: test_client.py
-      name: configure client
-  - test:
-      abort-on-fail: true
-      config:
-        command: add
-        id: client.4
-        node: node5
-        install_packages:
-          - ceph-common
-        copy_admin_keyring: true
-      desc: Configure the Cephfs client system 4
-      destroy-cluster: false
-      module: test_client.py
-      name: configure client
   - test:
         name: cephfs-volume-management
         module: cephfs_volume_management.py
         polarion-id: CEPH-83573446
         desc: cephfs volume management
         abort-on-fail: false
+  - test:
+      name: cephfs-snapshot-management
+      module: cephfs_snapshot_management.py
+      polarion-id: CEPH-83571366
+      desc: cephfs snapshot management
+      abort-on-fail: false
```

Comment:
It is a good practice to end with a newline character.

Reply:
I have added the extra line Satya